build.gradle

```diff
@@ -0,0 +1,50 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        kotlin_version = '1.3.72'
+        nav_version = '2.3.0-alpha05'
+        detekt_version = '1.7.4'
+    }
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        def gradle_plugin_version = '3.6.2'
+        classpath "com.android.tools.build:gradle:$gradle_plugin_version"
+
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$nav_version"
+
+        // Firebase
+        def google_services_plugin_version = '4.3.3'
+        classpath "com.google.gms:google-services:$google_services_plugin_version"
+        def firebase_crashlytics_plugin_version = '2.0.0'
+        classpath "com.google.firebase:firebase-crashlytics-gradle:$firebase_crashlytics_plugin_version"
+        def firebase_perf_plugin_version = '1.3.1'
+        classpath "com.google.firebase:perf-plugin:$firebase_perf_plugin_version"
```

Comment:
Following variables are used once:

- `google_services_plugin_version` 
- `firebase_crashlytics_plugin_version`
- `firebase_perf_plugin_version`

why do you define these values as variables?

Reply:
Somehow...
Fix it.